README.md

```diff
@@ -144,6 +144,43 @@ Before submitting a patch or a pull request make sure all tests are
 passing and that your patch is in line with the [contribution
 guidelines](CONTRIBUTING.md).
 
+### Working with mranderson (inlining dependencies)
+
+[mranderson](https://github.com/benedekfazekas/mranderson) is used to avoid classpath collisions.
+
+To work with `mranderson` the first thing to do is:
+
+`lein do clean, source-deps`
+
+this creates the munged local dependencies in target/srcdeps directory
```

Comment:
this -> This

Add the missing `.` to the sentence.

Wrap `target/srcdeps` in backticks.